.github/workflows/mayhem.yml

```diff
@@ -0,0 +1,74 @@
+name: Mayhem
```

Comment:
Can you please use the latest workflow example here: https://github.com/mayhemheroes/mayhem-cmake-example/blob/solution/.github/workflows/mayhem.yml